src/main/java/com/opentok/OpenTok.java

```diff
@@ -417,6 +417,10 @@ public Archive startArchive(String sessionId, ArchiveProperties properties) thro
         if (sessionId == null || sessionId == "") {
             throw new InvalidArgumentException("Session not valid");
         }
+        Boolean hasResolution =  properties != null && properties.resolution() != null && !properties.resolution().isEmpty();
+        if(properties != null && properties.outputMode().equals(Archive.OutputMode.INDIVIDUAL) && hasResolution) {
+            throw new InvalidArgumentException("Resolution must not be specified in individual output mode.");
```

Comment:
Could you please change the text to:

`The resolution cannot be specified for individual output mode.`